plugin-gradle/README.md

```diff
@@ -139,10 +139,10 @@ To apply freshmark to all of the `.md` files in your project, with all of your p
 ```gradle
 spotless {
 	freshmark {
-		target 'README.md', 'CONTRIBUTING.md'	// defaults to '**/*.md'
-		propertiesFile('gradle.properties')		// loads all the properties in the given file
+		target 'README.md', 'CONTRIBUTING.md'			// defaults to '**/*.md'
+		propertiesFile 'fresh.xml', 'mark.properties'	// loads all the properties in the given files
 		properties {
-			it.put('key', 'value')				// specify other properties manually
+			it.put('key', 'value')						// specify other properties manually
```

Comment:
> ...where for the other comments also tabs were used.

Really? I've just checked that section in the current live `plugin-gradle/README.md`, and it looks to me the comments are indented with spaces rather than tabs.

Could you stick with spaces for consistency, and make sure that all comments that _were_ aligned with each other before, continue to be aligned, please. :)